### ISSUE TITLE
A11y_Azure Tools for VS Code_Azure Function_Select a location for new resources_Voiceover: When the voiceover focus moves to listed items inside 'Recommended' and 'Other', the listed item numbers are being announced incorrectly. For example: VO announces, "East US" under Recommended as '2 of 25' instead of '1 of 25'.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/vscode-azurefunctions#3159]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsforVSCodeMAC_May2022;#Azure Tools for VS Code;#MAC;#FTP;#DesktopApp;#Voiceover;#WCAG1.3.1;#AINotInScope;#Closed;

## Environment Details:

Application Name: Azure Tools for VS Code
Application Version: 1.66.2 (Universal)
Mac Version: macOS Monterey 12.3.1
Screen Reader: Voiceover

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Visual Studio Code.
3. Click on extension button and search for "Azure Functions".
4. Click on install and tab to the "Azure" button.
5. Tab till 'Resource Groups' button and hit enter.
6. Tab till 'Create resource group' button and hit enter.
7. Tab till 'Select a location for new resources'.
8. Tab till 'Recommended' and expand.
9. Use Voiceover to observe.

## Actual:
When the voiceover focus moves to listed items inside 'Recommended' and 'Other', the listed item numbers being announced incorrectly or not in a proper hierarchy. For example: VO announces, "East US" under Recommended as '2 of 25' instead of '1 of 25'.

## Expected:
When the voiceover focus moves to listed items inside 'Recommended' and 'Other', the listed item numbers should be announced correctly. For example: VO should announce, "East US" under Recommended as '1 of 25' instead of '2 of 25'.

## User Impact:
Screen reader users will face difficulty in perceiving user interface component if incorrect listed numbers will be announced.

## Attachment:
![4_A11y_Visual Studio Code for MAC_Azure Function_Select a location for new resources_Voicover](https://user-images.githubusercontent.com/98393966/167088498-d575ccad-6610-4c60-be92-26abee3637b1.gif)

**Comment by MicroFish91** [comment]
**Note: Discussing if it makes sense to migrate to VS Code's Quick Pick Separators to circumvent the item numbering issue.

**Comment by MicroFish91** [comment]
Fixed by https://github.com/microsoft/vscode-azuretools/pull/1165

**Comment by bbarik06** [comment]
@MicroFish91 Please provide the latest environment where the fixes were made.

**Comment by MicroFish91** [comment]
https://dev.azure.com/ms-azuretools/534e1e1c-b364-44d0-b5ee-dce879a5106e/_apis/build/builds/48035/artifacts?artifactName=vsix&api-version=7.0&%24format=zip

**Comment by bbarik06** [comment]
Issue is repro. Please refer the attachment.
Version: 1.68.1
![3259](https://user-images.githubusercontent.com/98393966/175535313-508093e4-837e-407a-ad69-507269e6048c.gif)

**Comment by MicroFish91** [comment]
Hi @bbarik06, which version of Azure Functions are you running?  This is how it looks for me when I install the new .vsix.  This includes the latest VS Code version, see screenshot below:

![image](https://user-images.githubusercontent.com/40250218/175617339-43c6d795-3895-4aac-a811-c37d69f099a2.png)

Side note:  If the old link didn't work, try this one instead:  https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=48419&view=artifacts&pathAsName=false&type=publishedArtifacts

**Comment by Shivamd1** [comment]
issue not repro closing the bug
<img width="1540" alt="Screenshot 2022-07-20 at 1 18 38 PM" src="https://user-images.githubusercontent.com/60880975/179927195-bd042c64-580e-432f-ae92-deec9294578b.png">
